http/swagger.yml

```diff
@@ -7269,6 +7269,10 @@ components:
           readOnly: true
         retentionRules:
           $ref: "#/components/schemas/RetentionRules"
+        shardGroupDuration:
+          type: integer
+          format: int64
+          description: shard duration measured in nanoseconds
```

Comment:
Instead of nanos, could we use seconds to match the units used by `retentionRules`?